### PR TITLE
Several edits to readme.md

### PR DESCRIPTION
**Pull request by YamilVidal**

### CODE REVIEW

**Comment by pravsripad** [comment]
@YamilVidal should I go ahead and merge this or are you planning to make more changes?

**Comment by YamilVidal** [comment]
There will be more edits probably

Dr. Yamil Vidal
Predictive Brain Lab
Centre for Cognitive Neuroimaging - Donders Institute
Nijmegen
the Netherlands

**Comment by YamilVidal** [comment]
Hi @pravsripad 
At this point the Yamil branch contains several commits to be merged, and I'll be on holidays during the next week.
Could you merge it to main?
Thx!

File: coglib/fmri/readme.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Setup of data processing: Run only once during setup (requires sample dataset).
     bidsmapper ./temp_raw_for_bidscoin ./bids
 ```
 
+Run time ~= ???
```
Comment (david-rtr) [inline coglib/fmri/readme.md]: run time bids coin setup is a few seconds

File: coglib/fmri/readme.md [hunk 2]
```diff
@@ -75,18 +76,20 @@ Converts DICOM to BIDS compliant niftis.
     python 01_convert_dicom_to_bids.py
 ```
 
+Run time ~= ???
```
Comment (david-rtr) [inline coglib/fmri/readme.md]: dicom to bids probably 10min per subject

File: coglib/fmri/readme.md [hunk 3]
```diff
@@ -109,6 +113,7 @@ Run MRI QC for visual inspection of (f)MRI data quality. Perform visual inspecti
 	mriqc_sub.py ./bids -t 48 -w ./scratch/mriqc_workdir -o ./bids/derivatives
 ```
 
+Run time ~= ???
```
Comment (david-rtr) [inline coglib/fmri/readme.md]: mri qc maybe 1h

File: coglib/fmri/readme.md [hunk 4]
```diff
@@ -131,6 +137,7 @@ Extract IQMs of interest from MRI QC and reject runs/participants from further a
 	python 01_analyze_MRIQC_IQMs.py
 ```
 
+Run time ~= ???
```
Comment (david-rtr) [inline coglib/fmri/readme.md]: rejection is a few seconds

File: coglib/fmri/readme.md [hunk 5]
```diff
@@ -142,6 +149,7 @@ Preprocess (f)MRI data. Perform visual inspection of each runs data (see `./bids
 	fmriprep_sub.py ./bids -w ./scratch/fmriprep_workdir --time 80 --mem_mb 30000 -n 6 -a " --ignore sbref slicetiming --output-spaces T1w MNI152NLin2009cAsym"
 ```
 
+Run time ~= 2 days per subject
```
Comment (david-rtr) [inline coglib/fmri/readme.md]: prep is at most 2 days, when the HPC had some hiccups. more realistic is probably 1 day

**Comment by nbonacchi** [comment]
Is this safe to merge now?

**Review by nbonacchi — APPROVED** [review]
@david-rtr ?

**Pull request merged**